sql/report_queries/erm/count/erm_inventory_count.sql

```diff
@@ -0,0 +1,231 @@
+        ''::VARCHAR AS campus_filter, -- select 'Main Campus','City Campus','Online' as needed
+        ''::VARCHAR AS library_filter, -- select 'Datalogisk Institut','Adelaide Library' as needed
+        ''::VARCHAR AS institution_filter -- select 'Kobenhavns Universitet','Montoya College' as needed
+)
+SELECT
+    inst.type_id,
+    inst.type_name,
+    inst.mode_of_issuance_id,
+    inst.mode_of_issuance_name,
+    inform.format_id AS instance_format_id,
+    inform.format_code AS instance_format_code,
+    inform.format_name AS instance_format_name,
+    lng.language AS first_language,
+    insc.statistical_code_id AS instance_statistical_code_id,
+    insc.statistical_code AS instance_statistical_code,
+    insc.statistical_code_name AS instance_statistical_code_name,
+    innc.nature_of_content_term_id AS nature_of_content_id,
+    innc.nature_of_content_term_id AS nature_of_content_code,
+    innc.nature_of_content_term_name AS nature_of_content_name,
+    hld.type_id AS holdings_type_id,
+    hld.type_name AS holdings_type_name,
+    hld.call_number_type_id AS holdings_callnumber_type_id,
+    hld.call_number_type_name AS holdings_callnumber_type_name,
+    hsc.statistical_code_id AS holdings_statistical_code_id,
+    hsc.statistical_code AS holdings_statistical_code,
+    hsc.statistical_code_name AS holdings_statistical_code_name,
+    hld.receipt_status AS holdings_receipt_status,
+    loc.location_name,
+    inst.previously_held,
+    super_relation.relationship_type_id AS instance_super_relation_relationship_type_id,
+    super_relation.relationship_type_name AS instance_super_relation_relationship_type_name,
+    sub_relation.relationship_type_id AS instance_sub_relation_relationship_type_id,
+    sub_relation.relationship_type_name AS instance_sub_relation_relationship_type_name,
+    COUNT(DISTINCT inst.instance_id) AS "title count"
+FROM
+    folio_reporting.instance_ext AS inst
+        LEFT JOIN folio_reporting.instance_statistical_codes AS insc ON inst.instance_id = insc.instance_id
+        LEFT JOIN folio_reporting.instance_nature_content AS innc ON inst.instance_id = innc.instance_id
+        LEFT JOIN folio_reporting.instance_formats AS inform ON inst.instance_id = inform.instance_id
+        LEFT JOIN folio_reporting.instance_relationships_ext AS super_relation ON super_relation.relationship_super_instance_id = inst.instance_id
+        LEFT JOIN folio_reporting.instance_relationships_ext AS sub_relation ON sub_relation.relationship_sub_instance_id = inst.instance_id
+        LEFT JOIN folio_reporting.holdings_ext AS hld ON inst.instance_id = hld.instance_id
+        LEFT JOIN folio_reporting.holdings_statistical_codes AS hsc ON hld.holdings_id = hsc.holdings_id
+        LEFT JOIN folio_reporting.locations_libraries AS loc ON hld.permanent_location_id = loc.location_id
+        LEFT JOIN folio_reporting.instance_languages AS lng ON lng.instance_id = inst.instance_id AND lng.ordinality = 1
```

Comment:
I think if you merge upstream changes from main, then `lng.ordinality` should be `lng.language_ordinality`.